actions/detect-secrets/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Do we need all this licence stuff?

Reply:
And do we need this detect-secrets action? Seems tangential to semver.

Reply:
this is a publicly available function so seemed prudent to include a license file